Doc.md

```diff
@@ -0,0 +1,19 @@
+# Observation On Approaches To Flutter.
+
+## Using/Calling for an Image widget.
+
+We use the following code snippet when using an image widget after creating an image folder with the image needed in it, and setting the image evnironment at "Pubspec.yaml"...
+
+body: Center(
+        child: Image.asset('asset/image.png'),
+      ),

```

Comment:
Format to look like code @Oyebliss 
Use 4 backticks directly on top and directly below the code to achieve that.